### PR TITLE
Add Command to Manager to create app environment

### PR DESCRIPTION
**Pull request by ligthyear**
Also: updates the docs, reflecting this and the previous
changes to a vagrant based system.

### CODE REVIEW

File: docs/Development-App-Setup.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+
+=== Start your app in vagrant
+
+From within your vagrant type `./start.sh` to start the beavy processes.
+
+This will open a tmux splitted window session. On the top is a default
+shell for you to use (on vagrant of course).
+
+On the bottom left, you have first npm update and install all dependencies, then start webpack to compile the javascript files and automatically reload and compile them on change.
```
Comment (ezmiller) [inline docs/Development-App-Setup.adoc]: Maybe instead: "On the bottom left, `npm install` has been run to install all the dependencies, followed by webpack to compile all the javascript files and watch for changes. So when you make changes to a javascript file, webpack will automatically reload and compile the changes here."

File: docs/Development-App-Setup.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+On the bottom left, the session first installs all dependencies (depending on the modules you have configured), then run any migrations and start the flask process.
```
Comment (ezmiller) [inline docs/Development-App-Setup.adoc]: "...and then runs...and starts..."

**Comment by ezmiller** [comment]
@homu r+

**Comment by ligthyear** [comment]
@homu retry

**Comment by ligthyear** [comment]
@homu retry

**Pull request merged**